### PR TITLE
Vulture fixes

### PR DESCRIPTION
**Pull request by dgzlopes**
**What this PR does**:
- Fix goroutine leak by re-using the Jaeger gRPC client (moving it outside of the write loop).
- Change bucket name (from `failed` to `notfound`)

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by joe-elliott — APPROVED** [review]
might be easier to wrap this in function and use defer...but no big deal either way

File: cmd/tempo-vulture/main.go [hunk 1]
```diff
@@ -65,17 +65,16 @@ func main() {
 
 	// Write
 	go func() {
+		c, err := newJaegerGRPCClient(tempoPushURL)
+		if err != nil {
+			glog.Error("error creating grpc client", err)
+			metricErrorTotal.Inc()
```
Comment (dgzlopes) [inline cmd/tempo-vulture/main.go:71]: Well, if this happens, Vulture won't work... Idk if this is the best way to handle this...

Actually, It's going to crash.

Comment (dgzlopes) [reply cmd/tempo-vulture/main.go:71]: For example, if the tempoPushURL is malformed.

Comment (dgzlopes) [reply cmd/tempo-vulture/main.go]: Replaced this with panic...

**Review by joe-elliott — APPROVED** [review]
thanks!

**Pull request merged**